src/client/Microsoft.Identity.Client/AppConfig/TokenRequestContentType.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// The media content type for post requests made to the identity provider.
+    /// </summary>
+    public class TokenRequestContentType
+    {
+        internal string _value;
+
+        internal TokenRequestContentType(string value) { _value = value; }
+
+        /// <summary>
+        /// JSON media content type
+        /// </summary>
+        public static TokenRequestContentType JSON { get { return new TokenRequestContentType("application/json"); } }
```

Comment:
I think you should add an option for x-www-form-url which is the default.

https://stackoverflow.com/questions/9870523/what-are-the-differences-between-application-json-and-application-x-www-form-url